src/CLICommand/cli_command.ts

```diff
@@ -47,8 +47,10 @@ function setCommanderCommand(commandDescriptor: CommandDescriptor, program: CliA
 		}
 	});
 	command = command.action(async (options) => {
-		assertConjunctionParameters(commandDescriptor, options);
-		await commandDescriptor.action(options).catch((err) => {
+		await (async function () {
```

Comment:
So this will now print the error from `assertConjunctionParameters` rather than a call stack? Or whats before and after behavior change here?

Reply:
It will only print the error message, yes. Previously it thrown an uncaught exception.

I think we could make it to print the whole stack if there's the `--verbose` flag set :thinking:

Reply:
This fixes an issue Agustin is running into. Looks good to me

Sorry for the confusing question. I was just wondering if I was missing something here